agent/bench-scripts/pbench-uperf

```diff
@@ -416,47 +426,49 @@ if [[ "${postprocess_only}" != "y" ]]; then
 	for vserver in ${vsock_servers//,/ }; do
 		(( v_cnt++ ))
 	done
-	if [[ ${has_vsock} -eq 1 && ${v_cnt} -eq 0 ]]; then
-		error_log "You must provide vsock-servers when the vsock protocol is specified"
-		exit 1
-	fi
-	if [[ ${v_cnt} -gt 0 && ${v_cnt} -ne ${s_cnt} ]]; then
-		error_log "Number of servers and vsock-servers specified on command line must match"
-		error_log "    servers(${s_cnt}): ${servers}"
-		error_log "    vsock-servers(${v_cnt}): ${vsock_servers}"
+	if [[ ${has_vsock} -eq 1 ]]; then
+		if [[ ${v_cnt} -ne ${s_cnt} ]]; then
+			error_log "When using the vsock protocol, the number of servers and vsock-servers specified on command line must also match"
```

Comment:
I would drop "also".  And, given the context, I think we can drop "When using the vsock protocol", as well.  (And, we can probably dispense with "on command line", too.)  And, the result will fit in an 80-column line:
```
12345678901234567890123456789012345678901234567890123456789012345678901234567890
The number of servers and vsock-servers specified must match.
```
🙂